harmony_modules/controls.py

```diff
@@ -173,6 +178,33 @@ def deactivate(self):
         if self.nonverbal_gui_id is not None:
             self.toggle_nonverbal_actions(self.game)
 
+    def draw_interaction_target_selector(self):
+        self.chat_gui_data.target_selector = ComboListBox(
```

Comment:
please initialize this combo box only once. if you re-create combo box every time on paint, it cannot remember the selection from user.